scripts/wait-for-pg.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
We've used dockerize in the past for this kind of thing
https://github.com/jwilder/dockerize#waiting-for-other-dependencies

Reply:
I use `wget --output-document /dev/null --waitretry=5 --tries=10 --retry-connrefused localhost:4200 || true` because I likely don't have to install wget

Reply:
Hmm, so it seems waiting for tcp://5432 is not enough; if I don't have the sleep, I get an error on the psql statement -- I guess the port is open but PG hasn't finished initializing.

```
./dockerize -wait tcp://localhost:5432 -timeout 1m
#sleep 5
psql -h localhost -c '\dt;' -U postgres
psql: server closed the connection unexpectedly
	This probably means the server terminated abnormally
	before or while processing the request.
```